.github/workflows/docker-publish.yml

```
@@ -0,0 +1,68 @@
+
+      - name: Build Docker image
+        run: |
+          VERSION=${{ github.event.release.tag_name || 'v0.1.0' }}
+          docker build \
+            --build-arg VERSION=${VERSION} \
+            -t ghcr.io/rshade/finfocus-plugin-aws-public:latest \
+            -t ghcr.io/rshade/finfocus-plugin-aws-public:${VERSION} \
+            -f docker/Dockerfile .
+
+      - name: Check image size
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2079:error:2:13: (( )) doesn't support decimals. Use bc or awk

(shellcheck)